nav_msgs/OccupancyGrid map
---
nav_msgs/OccupancyGrid cleaned_map